### DISCUSSION TITLE
Liquid soap is not compiling on debian 12 due to error i think its debianutils issue when installing centovacast

### DISCUSSION THREAD

**Discussion by gothicdude1044** [savonet/liquidsoap#3579] [User Support]
**Describe the bug**
A clear and concise description of what the bug is. Add some script and log extract so we can see what is happening. Don't forget to add some context although not necessarily your whole script..

**To Reproduce**
A minimal script to reproduce the issue is the most efficient way to have your bug addressed and fixed very quickly!

**Expected behavior**

**Version details**

- OS: [e.g. Debian, OSX]
- Version [e.g. `1.3.4`]

**Install method**
Did you install via `opam`? Using your distribution's packages? From source?

**Common issues**
Can't decode or encode in some audio format? Check if you have [installed the correct dependencies](https://www.liquidsoap.info/#download)

**Comment by gothicdude1044** [comment]
Fatal error: exception Sys_error("Broken pipe")

<><> Updating package repositories ><><><><><><><><><><><><><><><><><><><><><><>
[default] synchronised from https://opam.ocaml.org
Now run 'opam upgrade' to apply any package updates.
O caml, my caml!
[NOTE] Package opam-depext is already installed (current version is 1.2.1-1).
Installing OS package dependencies ...
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package libfdk-aac-dev
! Error: Failed to install dependencies: debianutils
ladspa-sdk
libfdk-aac-dev
libflac-dev
libmad0-dev
libmagic-dev
libmp3lame-dev
libogg-dev
libpcre3-dev
libsamplerate0-dev
libtag1-dev
libvorbis-dev
pkg-config
zlib1g-dev
Liquidsoap installation failed
! Error: Installer exited with error, aborting

**Comment by vitoyucepi** [comment]
Hi @gothicdude1044,
The `libfdk-aac-dev` package comes from the `non-free` package archive.
You should modify `/etc/apt/sources.list.d/debian.sources`.
```diff
--- a/etc/apt/sources.list.d/debian.sources
+++ b/etc/apt/sources.list.d/debian.sources
@@ -2,12 +2,12 @@ Types: deb
 # http://snapshot.debian.org/archive/debian/20231120T000000Z
 URIs: http://deb.debian.org/debian
 Suites: bookworm bookworm-updates
-Components: main
+Components: main contrib non-free
 Signed-By: /usr/share/keyrings/debian-archive-keyring.gpg
 
 Types: deb
 # http://snapshot.debian.org/archive/debian-security/20231120T000000Z
 URIs: http://deb.debian.org/debian-security
 Suites: bookworm-security
-Components: main
+Components: main contrib non-free
 Signed-By: /usr/share/keyrings/debian-archive-keyring.gpg
```
Then run `apt-get update` to load new package lists into the local cache.
